package.json

```diff
@@ -22,7 +22,7 @@
     "@apollographql/graphql-playground-html": "1.6.24",
     "@parse/fs-files-adapter": "1.0.1",
     "@parse/push-adapter": "3.2.0",
-    "@parse/s3-files-adapter": "1.3.0",
+    "@parse/s3-files-adapter": "1.4.0",
```

Comment:
So who gets to update https://www.npmjs.com/package/parse-server and shouldn't we at least do a minor version number bump to 3.9.1 or 3.10.0?